server.js

```diff
@@ -143,6 +143,10 @@ app.use(express.static(path.join(__dirname, 'public')));
 app.use('/MathJax', express.static(path.join(__dirname, 'node_modules', 'mathjax')));
 app.use('/node_modules', express.static(path.join(__dirname, 'node_modules')));
 
+// Support legacy use of ace by v2 questions
+app.use('/public/localscripts/calculationQuestion/ace', express.static(path.join(__dirname, 'node_modules/ace-builds/src-min-noconflict')));
```

Comment:
Does this work? It looks to me like this route will be preemptively handled by the static route on line 142.

Reply:
Or does express static fall through if the file isn’t there? That would be both neat and slightly scary.

Reply:
I've tested one of the courses in question locally - seems to work fine.